en/importer/materials/fbx-materials.md

```diff
@@ -0,0 +1,122 @@
+# FBX Smart Material Conversion
+FBX Smart Material Conversion in Importer is a feature of model importer, which can map some standard materials in DCC tools to Cocos Creator's built-in materials, and restore the effects of materials in DCC tools.
+* The following materials are supported:
+
+|         | Phong               | PBR               |
+|---------|---------------------|-------------------|
+| 3ds Max | Standard(legacy)    | Physical Material |
+| Blender | None                | Principled BSDF   |
+| C4D     | Standard            | None              |
+| Maya    | Lambert/Blinn/Phong | Standard Surface  |
+ 
+* Here is the comparison of maya standard surface in Maya and Cocos Creator:
+
+| Maya Viewport              | Cocos Creator Viewport       |
+|----------------------------|------------------------------|
+| ![Maya](maya-viewport.png) | ![cocos](cocos-viewport.png) |
+
+You can refer to the following project files to ensure the importer can import Maya Standard Surface correctly.
+ [Maya Car Demo](maya_car.zip)
+
+## FBX Smart Material Conversion Importer Configuration and Process
```

Comment:
Smart Material Conversion configuration and import process for FBX assets

Reply:
Done